### PR TITLE
EZP-29101: Implement permissions for Content Types

### PR DESCRIPTION
**Pull request by mikadamczyk**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-29101
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    |no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

screen shot 2018-06-28 at 3 36 18 pm

screen shot 2018-06-28 at 3 36 38 pm

screen shot 2018-06-28 at 3 36 46 pm

screen shot 2018-06-28 at 3 37 09 pm

screen shot 2018-06-28 at 3 37 20 pm

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/admin/content_type_group/list.html.twig [hunk 1]
```diff
@@ -79,17 +94,19 @@
...
-                        </a>
+                        {% if can_update %}
+                            {% set edit_url = path('ezplatform.content_type_group.update', {
+                                contentTypeGroupId: content_type_group.id
+                            }) %}
+                            <a
+                                title="{{ 'content_type_group.view.list.action.edit'|trans|desc('Edit') }}"
+                                href="{{ edit_url }}"
+                                class="btn btn-icon mx-3">
+                                <svg class="ez-icon ez-icon-edit">
```
[the remark this answers is not in the document]
Comment (mikadamczyk) [reply src/bundle/Resources/views/admin/content_type_group/list.html.twig]: @sunpietro fixed

File: src/bundle/Resources/views/admin/content_type/list.html.twig [hunk 2]
```diff
@@ -8,27 +8,38 @@
...
-                'message': 'content_type.modal.message'|trans|desc('Do you want to delete Content Type?'),
-                'data_click': '#content_types_delete_delete',
-            }%}
+            {% if can_create %}
+                <a
+                    href="{{ path('ezplatform.content_type.add', {contentTypeGroupId: content_type_group.id}) }}"
+                    title="{{ 'content_type.view.list.action.add'|trans|desc('Create a Content Type') }}"
+                    class="btn btn-primary">
+                    <svg class="ez-icon ez-icon-create">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (sunpietro) [inline src/bundle/Resources/views/admin/content_type/list.html.twig]: This part: `xmlns:xlink="http://www.w3.org/1999/xlink"` is not needed at all.

Comment (sunpietro) [reply src/bundle/Resources/views/admin/content_type/list.html.twig]: Please, fix it in other places as well

Comment (mikadamczyk) [reply src/bundle/Resources/views/admin/content_type/list.html.twig]: @sunpietro fixed. As we discussed I will make another PR for other places.

**Pull request merged**